tests/mocked_plans/sample_plans/018346c5-f95c-46df-a34e-9a78bdec0018.yaml

```diff
@@ -453,6 +461,10 @@ plan: |
                 Enable nova-compute scheduler from unit: 'nova-compute-kvm/6'
                 Enable nova-compute scheduler from unit: 'nova-compute-kvm/7'
                 Enable nova-compute scheduler from unit: 'nova-compute-kvm/8'
+                Restart subordinate service for unit: 'ceilometer-agent/4'
+                Restart subordinate service for unit: 'ceilometer-agent/6'
+                Restart subordinate service for unit: 'ceilometer-agent/7'
+                Restart subordinate service for unit: 'ceilometer-agent/8'
```

Comment:
nitpick: I don't really understand the words here.
suggestion: `Restart service for subordinate unit: xxx`

Reply:
Updated